python/tk_desktop/project_model.py

```diff
@@ -214,6 +215,22 @@ class SgProjectModel(ShotgunModel):
     thumbnail_updated = QtCore.Signal(QtGui.QStandardItem)
     project_launched = QtCore.Signal()
 
+    _supports_project_templates = None
+
+    @staticmethod
+    def supports_project_templates(connection):
+        # If we haven't checked on the server yet.
```

Comment:
Doc string?  We should include the reason why we are doing this check (Template projects in 6.0) so whoever looks at this 5 years from now understands the motivation for the check.